css/main.css

```diff
@@ -552,93 +557,93 @@ input[type=text]{
 }
 
 /* -Contenedores de Secciones principales -Contenedores de Secciones principales -Contenedores de Secciones principales -Contenedores de Secciones principales*/
-div#divTienda{
+div#div-tienda {
     background-color: #F4F1EF;
 }
 
-div#divTienda img{
+div#div-tienda img {
     max-width: 100%;
-    width:100vw; 
+    width: 100vw; 
 }
 
 /*  -Media Querys -Media Querys -Media Querys -Media Querys -Media Querys -Media Querys -Media Querys -Media Querys -Media Querys -Media Querys -Media Querys -Media Querys -Media Querys */
 
-@media screen and (max-width:890px){
+@media screen and (max-width:890px) {
 
-    div#nav li a{        
+    div#nav li a {        
         margin-left: 15px;
         margin-top: 10px;
     }
 
-    div#divProductos #flexChild1 {
+    div#div-productos #flexChild1 {
         flex-direction: column;
     }
 
-    div#divProductos #productosLimpieza #imgArriba{
+    div#div-productos #productos-limpieza #img-arriba {
         display: none;
         padding-top: 10px;
     }
 
-    div#divProductos{
+    div#div-productos {
         width: 100%;
     }
 
-    div#divProductos #productosLimpieza {
+    div#div-productos #productos-limpieza {
         width: 100%;
         margin: 0px; 
 
     }
 
-    div#divProductos .botonLimpieza {
+    div#div-productos .boton-limpieza {
         padding-bottom: 10px;
         width: 100%;
         margin: 5px;
     }
 
-    button#btnMVV{
+    button#btn-mvv {
 
         /*aqui un comentario para crear algo diferente para el push*/
     }
 
-    button#btnMVV{
+    button#btn-mvv {
         width: 226px;
         padding-left: 20px;
         padding-right: 20px;
         text-align: center;
         font-size: 18px;
     }
 
-    div#nav li a{
+    div#nav li a {
         display: none;
     }
 
-    div#divContactos .entradasForm #divFlexForm img {
+    div#div-contactos .entradas-form #div-flex-form img {
```

Comment:
This seems duplicate, can you take a look?

Reply:
done